docs/examples/tutorial/string/to_unicode.pyx

```diff
@@ -1,15 +1,9 @@
-from cpython.version cimport PY_MAJOR_VERSION
-
 
```

Comment:
It's possible this blank line at the start might not display so the code might not align correctly - I'll have a quick check of the generated docs before proceeding